### PR TITLE
REST API + new status style

### PR DESCRIPTION
**Pull request by VStankoV**

### CODE REVIEW

File: data-model-api/src/main/java/com/provectus/prodobro/actor/company/Company.java [hunk 1]
```diff
@@ -25,15 +25,19 @@
...
      *
-     * @return {@link Set< Info >} of Actor`s information
+     * @return {@link Set<Info>} of Actor`s information
      */
     Set<Info> getInfo();
 
     void setInfo(Set<Info> info);
 
-    Status getStatus();
+    StatusEnumNew getStatus();
```
Comment (AiRmode) [inline data-model-api/src/main/java/com/provectus/prodobro/actor/company/Company.java]: Need to remane

File: data-model-api/src/main/java/com/provectus/prodobro/shared/StatusEnumNew.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
+package com.provectus.prodobro.shared;
+
+
+public enum StatusEnumNew {
```
Comment (AiRmode) [inline data-model-api/src/main/java/com/provectus/prodobro/shared/StatusEnumNew.java]: Please rename

**Review by AiRmode — CHANGES_REQUESTED** [review]
There are some conflicts, please resolve them first

**Comment by VStankoV** [comment]
conflicts was resolved by merging and enum was renamed

**Pull request closed without merging**